en/services/opendroneid.md

```diff
@@ -38,10 +39,108 @@ Message Pack | [OPEN_DRONE_ID_MESSAGE_PACK](../messages/common.md#OPEN_DRONE_ID_
+Receivers should only listen to messages that have these fields set to either zero (broadcast) or the receivers own system ID and/or component ID.
+This can be useful if e.g. there are two UA connected to a single GCS.
+The GCS can then direct information to specific MAV_COMP_ID_ODID_TXRX_x components on a specific UA.
+By default, all senders of drone ID messages should fill the `target_system` and `target_component` fields with zero, to indicate a broadcast to all receivers.
+
+> **Note** WIP: Security of drone ID data is currently quite open and it is unclear if it will be required in some locations or for some use cases. 
+ Most likely the transmitter components will be the ones to generate the signature for the Authentication message but the signing mechanism and how the key(s) for this is managed is open.
+
+> **Note** WIP: How will the Internet tranciever be configured? It needs to know what server(s) to connect to, credentials etc.
+
+### Open Drone ID data from other UA
+It is possible that the transmitter components also work as receivers, for obtaining drone ID data from sourrounding UAs.
+When publishing the received drone ID data as internal MAVLink messages, they must set the `compid` field to their own MAV_COMP_ID_ODID_TXRX_n ID to make it possible to distinguish this data from the drone ID data of the UA itself.
+
+At least two possible consumers of drone ID data from sourrounding aircrafts are possible.
+- A Detect And Avoid (DAA) system will track the current and estimated future positions of the other UAs and take that into account when setting the fligt path of the UA itself.
+- A Remote ID Display (RID) application will visually show the sourrounding UA's locations (and possibly past and estimated future flight paths) to the operator of the UA, in order for him/her to utilize this information when controlling the UA.
+
+See below on how to combine data from other UAs.
+
+### Heartbeat
+Each component listed in the table above, is required to regularly send out MAVLink [HEARTBEAT](../messages/common.md#HEARTBEAT) messages in order to facilitate discovery and monitoring of the component in the UAS.
+For MAV_COMP_ID_ODID_TXRX_x components, the `type` field in the HEARTBEAT message must be filled with [MAV_TYPE_ODID](../messages/common.md#MAV_TYPE_ODID).
+Please see further details in the [Heartbeat documentation](heartbeat.md).
+ 
+
+## UAS with multiple transmitters and/or receivers
+
+Since three different methods of broadcasting/publishing drone ID data has been defined, it is quite possible and desirable for a UAS to have more than just a single type.
+Exact legislation for drone ID support in different regions is still in the definition phase but e.g. the current FAA rule proposal mandates that for certain categories of UA, both broadcast of it's ID via either Bluetooth or WiFi must be performed and also simultaneous transmission of it's ID via the internet to a Remote ID server must be performed.
```

Comment:
".. but e.g. ..." that sounds like a copy/paste mistake?

Reply:
I've tidied this text